src/index.ts

```diff
@@ -84,4 +87,13 @@ export = class extends Composer implements RNS {
   get subdomains(): Subdomains {
     return this._subdomains;
   }
+
+  /**
+   * Set of subdomains related methods
+   *
+   * @returns Object with subdomains related methods ready to use.
+   */
+  get utils(): Utils {
```

Comment:
Following comment on the class, this would
```javascript
return {
  namehash: utils.namehash,
  sha3: utils.sah3,
  // ...
}
```

Is compatible with `interface Utils` and is not a class.

Reply:
Completely agree

Reply:
🤝